misc/typing-inspect/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.7.1" %}
```

Comment:
we don't actually need this, since it's available on conda-forge:
https://anaconda.org/conda-forge/typing_inspect

Reply:
Done